mods/default/nodes.lua

```diff
@@ -2013,10 +2013,12 @@ function default.register_chest(name, d)
 			local meta = minetest.get_meta(pos)
 			meta:set_string("formspec", nil)
 			local inv = meta:get_inventory()
-			local list = inv:get_list("main")
-			inv:set_list("main", nil)
-			inv:set_size("default:chest", 8*4)
-			inv:set_list("default:chest", list)
+			local list = inv:get_list("default:chest")
+			if list then
+				inv:set_size("main", 8*4)
+				inv:set_list("main", list)
+				inv:set_list("default:chest", nil)
+			end
```

Comment:
I think it's fine. The `default:chest` inventory only existed for a short amount of time and never in a release. We can instruct people to reset the LBMs manually.